src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -592,8 +597,8 @@ def listFileBlockLocation(self, fileBlockName, dbsOnly = False, phedexNodes=Fals
                 msg += "%s\n" % formatEx3(ex)
                 raise DBSReaderError(msg)
 
-            if not any(blockInfo.values()): # no data location from dbs
-                return list()
+        if not any(blockInfo.values()): # no data location
```

Comment:
I don't want to do changes that are not necessary for the purpose of my PR (I do it in CRAB a lot, but I try to not do it in WMCore).
But since you tell me... I will remove `.values()`. This is assuming you can never have `blockInfo[blockname] = []`.